SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -326,22 +291,31 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	bool bGenerateDefaultLaunchConfig;
 
 	/** Returns which runtime variant we should use. */
-	TEnumAsByte<ESpatialOSRuntimeVariant::Type> GetSpatialOSRuntimeVariant() const { return RuntimeVariant; }
+	TEnumAsByte<ESpatialOSRuntimeVariant::Type> GetSpatialOSRuntimeVariant() const { return ESpatialOSRuntimeVariant::Standard; }
 
-	/** Returns the version information for the currently set variant*/
+	/** Returns the version information for the currently set runtime variant*/
 	const FRuntimeVariantVersion& GetSelectedRuntimeVariantVersion() const
 	{
-		return const_cast<USpatialGDKEditorSettings*>(this)->GetRuntimeVariantVersion(RuntimeVariant);
+		return const_cast<USpatialGDKEditorSettings*>(this)->GetRuntimeVariantVersion(ESpatialOSRuntimeVariant::Standard);
```

Comment:
same question: do we still need all this code now that we don't support both versions anymore?

Reply:
See above comment